src/pyodide-js/rollup.config.ts

```diff
@@ -0,0 +1,42 @@
+import typescript from 'rollup-plugin-typescript2';
+import replace from '@rollup/plugin-replace';
+
+const pkg = require("./package.json");
+
+const pyodideCdnUrl = process.env["PYODIDE_CDN_URL"] || `https://cdn.jsdelivr.net/pyodide/${pkg.version}/full/`;
+const pyodideAbiNumber = process.env["PYODIDE_ABI_NUMBER"] === undefined ? `1` : process.env["PYODIDE_CDN_URL"];
```

Comment:
```suggestion
const pyodideAbiNumber = process.env["PYODIDE_ABI_NUMBER"] === undefined ? `1` : process.env["PYODIDE_ABI_NUMBER"];
```
I think that would also fix the ABI version error.

Generally in terms of the name it would have been good to keep the same name as in the Makefile, `PYODIDE_PACKAGE_ABI` . Or else we need to change it quickly in the [FAQ](https://pyodide.readthedocs.io/en/latest/faq.html#how-to-detect-that-code-is-run-with-pyodide) before it gets carried over to various builds setups.

Reply:
Happy to change it to whatever you think makes sense :)